data/transition-sites/dclg_firekills.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: dclg_firekills
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
```

Comment:
It won't make any difference as this is a global 301, but this should probably just be the same as the global 301 because the old site was not about DCLG, it was about fire prevention policy.